### PR TITLE
Feature/extend pick functionality

### PR DESCRIPTION
**Pull request by dictor93**
### Description:
Extended pick feature functionality by adding multiple path arguments and allowing nested pick using string dot path notation or array path variant

### PR status:
- [x] Version bumped
- [x] Change-log updated
- [x] Tests added or updated
- [x] PR keeps 100% test coverage
- [x] Linter is applied
- [x] Const arrow functions are used instead of pure function definitions where applicable
- [x] Functions are listed in alphabetic order in index.js, index.d.ts and readme

### CODE REVIEW

**Comment by dictor93** [comment]
[Related PR](https://github.com/bitfinexcom/bfx-core-kyc-js/pull/1026)

**Review by jarco-develop — APPROVED** [review]
LGTM

File: test/pick.js [hunk 1]
```diff
@@ -29,4 +29,21 @@ describe('pick', () => {
   it('should handle keys that are not strings', () => {
     assert.deepStrictEqual(pick({ 1: 'one', 2: 'two' }, ['1', '2']), { 1: 'one', 2: 'two' })
   })
+
+  it('should pick properties by few path parameters', () => {
+    assert.deepStrictEqual(pick({ a: 1, b: 2, c: 3, d: 4 }, ['a', 'b'], 'd'), { a: 1, b: 2, d: 4 })
```
Comment (ezewer) [inline test/pick.js]: Can we split in shorter lines please?

**Review by ezewer — COMMENTED** [review]
Code looks good, split in lines for better reading.

**Review by ezewer — APPROVED** [review]
looks good to me!

**Review by grantfayvor — APPROVED** [review]
lgtm!

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# 1.7.1
+- feat: exrtended pick functionality by using multiple path parameters and nested pick
```
Comment (vigan-abd) [inline CHANGELOG]: typo

**Review by vigan-abd — CHANGES_REQUESTED** [review]

**Comment by vigan-abd** [comment]
also branch has conflicts

File: CHANGELOG [hunk 3]
```diff
@@ -1,3 +1,5 @@
+# 1.8.1
+- feat: extended pick functionality by using multiple path parameters and nested pick
```
Comment (vigan-abd) [inline CHANGELOG:2]: newline missing

**Review by vigan-abd — CHANGES_REQUESTED** [review]

**Review by vigan-abd — APPROVED** [review]

**Pull request merged**